src/map.c

```diff
@@ -44,3 +64,11 @@ Window *map_get(xcb_keysym_t keysym)
     LOG("selection not in range\n");
     return NULL;
 }
+
+void map_deinit()
```

Comment:
Can we rename this to `map_free`? Would be consistent with window, where we have `window_free`.

Reply:
done